src/shared/sam/debugger/commands/addSamDebugConfiguration.ts

```diff
@@ -127,7 +128,21 @@ export async function addSamDebugConfiguration(
             return
         }
     } else if (type === API_TARGET_TYPE) {
-        samDebugConfig = createApiAwsSamDebugConfig(workspaceFolder, runtimeName, resourceName, rootUri.fsPath)
+        // If the event has no properties, the default will be used
```

Comment:
How can you tell what the correct event to set is if you're passing a map of events here?